modules/distributor/receiver/middleware.go

```diff
@@ -56,8 +57,15 @@ func (m *multiTenancyMiddleware) Wrap(next consumer.Traces) consumer.Traces {
 		var err error
 		_, ctx, err = user.ExtractFromGRPCRequest(ctx)
 		if err != nil {
-			log.Logger.Log("msg", "failed to extract org id", "err", err)
-			return err
+			// Maybe its a HTTP request.
+			info := client.FromContext(ctx)
+			orgIDs := info.Metadata.Get(user.OrgIDHeaderName)
+			if len(orgIDs) != 1 {
+				log.Logger.Log("msg", "failed to extract org id", "err", err)
```

Comment:
can we add more detail to this error message? now it represents 2 errors in a row. failure to pull the orgid from grpc and failure to pull the orgid from http. perhaps we can log the value of `orgIDs` as well?